crates/ironrdp-rdpdr-native/src/nix/mod.rs

```diff
@@ -0,0 +1,2 @@
+#[allow(clippy::unnecessary_cast)]
```

Comment:
question: Why did you allow this warning? Could you add a comment explaining why directly in the code?

Reply:
because statvfs.blocks_available() return u32/u64 on different platform(for example, on my computer its u32), but volume_serial_number is u32, so i need to use ('as u32' or u32::try_from()),but it emit warning 'unnecessary_cast', so i add this

Reply:
ok, i added this explation in the newest commit. But when i change my code from 'as u32' to 'u32::try_from', it emit 'useless_conversion' warning, so i use '#[allow(clippy::useless_conversion)]' in the simple.rs and i add explanation.